packages/ffe-grid-react/src/InlineGrid.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import classNames from 'classnames';
+import { node, string } from 'prop-types';
+
+export default function InlineGrid(props) {
+    const { children, className, element, ...rest } = props;
+
+    const Element = element || 'div';
```

Comment:
Minor, men jeg syns egentlig det er penere å bruke `InlineGrid.defaultProps` for defaults.